app/views/privacy_policies/show.html.erb

```diff
@@ -0,0 +1,103 @@
+        <ul class="govuk-list govuk-list--bullet">
+          <li>your name</li>
+          <li>your phone number</li>
+          <li>your email address</li>
+          <li>where you work</li>
+          <li>what your role is</li>
+        </ul>
+
+  <h2 class="govuk-heading-m">How we use your data</h2>
+    <h3 class="govuk-heading-s">Processing applications</h3>
+      <p class="govuk-body">Processing applications includes the following:</p>
+        <ul class="govuk-list govuk-list--bullet">
+          <li>Sending applications to training providers</li>
+          <li>managing communications between candidates and training providers</li>
+          <li>working out and funding candidates are entitled to</li>
+          <li>getting in touch if there is a security issue around your data</li>
+          <li>getting in touch with candidates and training providers about their applications</li>
+        </ul>
+    <h3 class="govuk-heading-s">Building a better teacher training application process</h3>
+      <p class="govuk-body">For example, we'll look at any feedback you give us about our services so we can improve them.</p>
+      <p class="govuk-body">If you're an early career teacher or you work at a teacher training provider, we may contact you to carry out user research.</p>
+    <h3 class="govuk-heading-s">Getting insight to make government policy better</h3>
+      <p class="govuk-body">We might analyse your data to help us inform government policy around teacher recruitment and retention.</p>
+
+  <h2 class="govuk-heading-m">Who we share your data with</h2>
+    <p class="govuk-body">We do so under section 132 of the Education Act 2002, together with regulation 5 of the Education (School Teachers Qualifications) (England) Regulations 2003.</p>
+    <h3 class="govuk-heading-s">Sharing data with teacher training providers</h3>
+    <p class="govuk-body">We share early career teacher and mentor data with the teacher training providers the candidate applies to.</p>
+    <p class="govuk-body">We have a data sharing agreement with providers so they can only use your data to:</p>
+      <ul class="govuk-list govuk-list--bullet">
+        <li>process agreements, which may include contacting candidates</li>
+        <li>get statistics for internal use</li>
+        <li>get in touch with you if there is a security issue concerning your data</li>
+      </ul>
+
+  <h2 class="govuk-heading-m">External organisations who process data</h2>
+    <h3 class="govuk-heading-s">Customer service systems</h3>
+      <p class="govuk-body">We use a customer service management system to process some personal data, such as feedback you send us.</p>
+      <p class="govuk-body"><%= link_to 'Zendesk', "https://www.zendesk.co.uk/company/customers-partners/master-subscription-agreement/?cta=msa#confidentiality", class: "govuk-link" %> is currently our preferred customer service management system because it uses various safeguards to look after your data.</p>
```

Comment:
please use `govuk_link_to` instead of `link_to`

Reply:
Out of curiosity, what is difference between govuk_link_to and link_to with class: "govuk_link" ?